### DISCUSSION TITLE
Permanently scaling LoRA weights to be weaker?

### DISCUSSION THREAD

**Discussion by space-nuko** [kohya-ss/sd-scripts#298] [Ideas]
I don't know if this should be an issue instead, but here's my idea:

A lot of models on CivitAI are overbaked and work at lower weights like 0.5, but the model will work fine at that weight. So what if there were some way of scaling the weights in the model such that the 0.5 weight became equivalent to the 1.0 weight? Like a script you could call like `scale_weights.py /path/to/lora.safetensors --from 0.5 --to 1.0`

I don't know much about the math behind it, but is it doable in theory?

**Comment by weihaozhaung** [comment]
I don't know if you already found the solution,check out https://github.com/ostris/ai-toolkit, there's a lora weight scaler

**Comment by TingTingin** [comment]
You can merge the lora with an empty lora and a lower weight
